src/components/BallotCard.jsx

```diff
@@ -191,6 +192,18 @@ export class BallotCard extends React.Component {
         this.creator = fullName ? fullName : _miningKey;
     }
 
+    @action("validator has already voted")
+    getHasAlreadyVoted = async () => {
+        const { contractsStore, id, votingType } = this.props;
+        let _hasAlreadyVoted;
```

Comment:
As a minor note, `_hasAlreadyVoted` should be set to something by default (probably to `false` the same as in constructor)?

Reply:
you're right, thanks